pattern/variable/SystemReference.java

```diff
@@ -24,7 +24,7 @@
 
 public class SystemReference extends Reference.Name {
 
-    public static String PREFIX = "$/";
+    public static String PREFIX = "$/"; // TODO replace with GraqlToken.Char.$SYS
```

Comment:
Can you just replace it here anyways?

Reply:
turns out that this has been deleted from Graql, correctly, and should only live here - i was looking at an outdated Graql repo!